enterprise/frontend/src/embedding-sdk/components/private/SdkUsageProblem/SdkUsageProblemBanner.tsx

```diff
@@ -27,27 +19,25 @@ export interface SdkUsageProblemBannerProps {
 // so they remain legible even when the theme is changed.
 const unthemedBrand = originalColors["brand"];
 const unthemedTextDark = originalColors["text-dark"];
-const unthemedTextMedium = originalColors["text-medium"];
 
 export const SdkUsageProblemBanner = ({
   problem,
 }: SdkUsageProblemBannerProps) => {
-  const theme = useMantineTheme();
-
   const [expanded, setExpanded] = useState(false);
-  const [shown, { close: hideBanner }] = useDisclosure(true);
 
-  if (!problem || !shown) {
+  if (!problem) {
     return null;
   }
 
-  const { severity } = problem;
-  const isError = severity === "error";
+  const { title } = problem;
 
   // When the font family cannot be loaded from the MB instance,
   // due to MB instance outage or missing CORS header,
   // we fall back to the system font.
-  const fontFamily = `${theme.fontFamily}, sans-serif`;
+  const fontFamily = `${DEFAULT_FONT}, sans-serif`;
```

Comment:
We should actually use the default Metabase font (Lato), not the user's font family. We want to keep the Metabase branding as this is not shown to the user.